context/http.go

```diff
@@ -304,20 +293,10 @@ func (irw *instrumentedResponseWriter) Value(key interface{}) interface{} {
 			return irw
 		}
 
-		if !strings.HasPrefix(keyStr, "http.response.") {
-			goto fallback
-		}
-
-		parts := strings.Split(keyStr, ".")
-
-		if len(parts) != 3 {
-			goto fallback
-		}
-
 		irw.mu.Lock()
 		defer irw.mu.Unlock()
```

Comment:
I should point out that in my latest patch, we are obtaining a lock, even in the default (fallback) case; let me know if that's an issue